src/layout/section/faqSection/FaqSection.unit.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+
+import { render, screen } from '@testing-library/react'
+
+import { FaqItemProps, FaqSection, FaqSectionProps } from './index'
+
+const defaultItems: Array<FaqItemProps> = [
+  {
+    question: 'Question 1',
+    answer: `${'Answer to the question '.repeat(30)}`,
+  },
+  {
+    question: 'Question 2',
+    answer: 'A short answer.',
+  },
+  {
+    question: 'Question 3',
+    answer: `${'Answer to the question '.repeat(50)}`,
+  },
+  {
+    question: 'Question 4',
+    answer: `${'Another answer '.repeat(50)}`,
+  },
+]
+
+const minimalProps: FaqSectionProps = {
+  items: defaultItems,
+  expandLabel: 'Read more',
+}
+
+const allProps: FaqSectionProps = {
+  ...minimalProps,
+  title: 'section title',
+  buttonLabel: 'button label',
+  buttonHref: 'https://blablacar.com',
+}
+
+describe('FaqSection', () => {
+  it('should render FaqSection section with minimal props', () => {
+    render(<FaqSection {...minimalProps} />)
+
+    expect(screen.getByText('Question 1')).toBeInTheDocument()
```

Comment:
You can be a bit more precise with the semantic:

```suggestion
    expect(screen.getByRole('heading', { name: 'Question 1' })).toBeInTheDocument()
```

Reply:
Done